src/client/event_loop.rs

```diff
@@ -248,7 +249,7 @@ impl<T: Transport, L: LoginCredentials> ClientLoopWorker<T, L> {
     ///
     /// The client will make best attempts to stay joined to this channel. I/O errors will be
     /// compensated by retrying the join process. For this reason, this method returns no error.
-    fn join(&mut self, channel_login: String) {
```

Comment:
All of these kinds of changes, where the type in a struct or the argument type of a function changes, are breaking changes.

Reply:
Oh, looks to be missed. I've planned to rewrite it into `impl Into<FastStr>`, to save backwards compatibility.